packages/binding-wotfirestore-browser-bundle/package.json

```diff
@@ -2,7 +2,10 @@
   "name": "@node-wot/binding-wotfirestore-browser-bundle",
   "version": "0.8.0",
   "description": "A binding-wotfirestore bundle that can run in a web browser",
-  "repository": "https://github.com/hidetak/thingweb.node-wot",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hidetak/thingweb.node-wot.git"
+  },
```

Comment:
Why does repository look different here, we seem to use something like this elsewhere

"repository": "https://github.com/eclipse/thingweb.node-wot/tree/master/packages/binding-wotfirestore",